Formula/gst-validate.rb

```diff
@@ -49,6 +47,8 @@ def install
       system "make"
       system "make", "install"
     end
+
+    inreplace bin/"gst-validate-launcher", "#!/usr/bin/env python3", "#!#{Formula["python@3.8"].opt_bin}/python3"
```

Comment:
It doesn't matter much, but the recommended way is `rewrite_shebang detected_python_shebang, bin/"gst-validate-launcher"` (needs a `include Language::Python::Shebang` at the top of the formula).

Reply:
Oh and this one should probably have a revision bump.